### PR TITLE
Cleanup and Tactician version update

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,42 @@
...
-## Authors
+``` bash
+$ phpunit
+```
+
+## Credits
+
+- [Nigel Greenway](https://github.com/NigelGreenway)
+- [Ross Tuck](https://github.com/rosstuck)
+- [All Contributors](https://github.com/thephpleague/tactician-bernard/contributors)
```
Comment (rosstuck) [inline README.md]: Still says bernard. :)

File: composer.json [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+    "license":     "MIT",
+    "authors": [
+        {
+            "name": "Nigel Greenway",
+            "homepage": "http://futurepixels.co.uk"
+        }
+    ],
+    "require": {
+        "php": ">=5.5",
+        "league/tactician": "^0.5@dev",
```
Comment (rosstuck) [inline composer.json]: Don't think this needs the @dev flag anymore now that 0.5 is just about out the door

**Comment by rosstuck** [comment]
Just the two tweaks, then it's okay with me. @NigelGreenway, any thoughts? :)

File: README.md (hunk 1 above)
Comment (sagikazarmark) [inline README.md]: Ah, I am busted, copy&paste :stuck_out_tongue_winking_eye:

File: composer.json (hunk 2 above)
Comment (sagikazarmark) [inline composer.json]: I just removed that, but haven't pushed it yet.

**Comment by sagikazarmark** [comment]
After a few minutes tactician is released, tests should be ran again.

**Comment by NigelGreenway** [comment]
Sorry for the late reply.  All good my end from what can see.

**Comment by sagikazarmark** [comment]
@NigelGreenway I am not sure about why `addHandlers` was protected, but I made it public. Also, added some tests for add\* methods.

**Comment by NigelGreenway** [comment]
It was meant to be private,  my thought was so the mapping files are part of the initiation of the plug in. Is it generally best practice to have a configuration passed outside of the construction of the object?

**Comment by sagikazarmark** [comment]
I think - in case of locators - it is. Since it is some sort of command routing.

**Comment by sagikazarmark** [comment]
@rosstuck There is definitely something bad going on here. I've just got an email with the text 'Still says bernard. :)'. But I can't see it on github.

Also, it has been corrected in b423afed6beb89b903be4b6d4fd1ddd74d6f7f94

**Comment by sagikazarmark** [comment]
Ah, found it in an outdated diff. Seems like github has some delay (few hours??).

**Comment by sagikazarmark** [comment]
@rosstuck Ping

**Comment by rosstuck** [comment]
Thanks for the cleanup work. :)

**Pull request merged**